addons/website/views/snippets/s_product_list.xml

```diff
@@ -2,55 +2,56 @@
 <odoo>
 
 <template name="Items" id="s_product_list">
+    <t t-set="url" value="url or ''"/>
```

Comment:
- This whole commit should be moved in a PR for master of the same task, no reason to be in 14.0, this is a pure IMP.
- It should be "#" as fallback instead of an empty string